clusterloader2/run-e2e.sh

```diff
@@ -22,5 +22,17 @@ CLUSTERLOADER_ROOT=$(dirname "${BASH_SOURCE}")
 export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
+# Deploy the GCP PD CSI Driver is required
+if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
+   if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
+      echo "Env var E2E_GOOGLE_APPLICATION_CREDENTIALS must be set to deploy driver"
+      exit 1
+   else
```

Comment:
remove else - there is exit 1 above anyway

Reply:
ack